src/styles/LessonSide.scss

```
@@ -1,7 +1,69 @@
 @use '_colors.scss' as colors;
 
+@mixin font-styles {
+  font-family: Quattrocento Sans, sans-serif;
+  font-size: 30px;
+  font-style: normal;
+  font-weight: bold;
+}
+
 #lesson-side-container {
   background-color: colors.$primary-green;
   height: 100%;
+  padding: 20px;
   width: 40%;
 }
+
```

Comment to easily separate where css is for
```suggestion

// LevelSelector 
```